packages/api/src/external/commonwell/document/document-query-sandbox.ts

```diff
@@ -207,5 +228,15 @@ function addSandboxFields(docRef: DocumentReference): DocumentReference {
     }
   }
 
+  if (!docRef.date) {
+    docRef.date = docRef.content?.[0]?.attachment?.creation;
+  }
+
+  docRef.contained?.push({
+    resourceType: "Organization",
+    id: "Sandbox example org",
+    name: `Hospital org#${Math.floor(Math.random() * 1000)}`,
+  });
```

Comment:
Shouldn't this the the cx's org?

Reply:
no, these imitate outside organizations that created and posted the document